app/components/reaction_picker/reaction_picker.js

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {PureComponent} from 'react';
+import PropTypes from 'prop-types';
+import {
+    Image,
+    View,
+    TouchableWithoutFeedback,
+} from 'react-native';
+
+import Emoji from 'app/components/emoji';
+import {changeOpacity, makeStyleSheetFromTheme} from 'app/utils/theme';
+import {hapticFeedback} from 'app/utils/general';
+import addReactionIcon from 'assets/images/icons/reaction.png';
+import {REACTION_PICKER_HEIGHT} from '../../screens/post_options/post_options_utils';
```

Comment:
```suggestion
import {REACTION_PICKER_HEIGHT} from 'app/screens/post_options/post_options_utils';
```

And maybe consider adding the constant in the constant folder